LongWarOfTheChosen/Localization/LW_PerkPack_Integrated/XComGame.int

```diff
@@ -692,7 +692,7 @@ LocFriendlyName="Body Shield"
 LocFlyOverText="Body Shield"
 LocLongDescription="A targeted enemy receives -20 aim and -50 critical chance against the soldier."
 LocHelpText="The targeted enemy receives -20 aim and -50 critical chance against the soldier."
-LocPromotionPopupText="<Bullet/> A targeted enemy receives -20 aim and -50 critical chance against the soldier.<br/>"
+LocPromotionPopupText="<Bullet/> A targeted enemy receives -20 aim and -50 critical chance against the soldier.<br/><Bullet/> Not cost an action.<br/><Bullet/> Body Shield has a 2 turn cooldown."
```

Comment:
Rather than "Not cost an action", use "This is a free action". Also, see if `<Ability:SelfCooldown/>` works in place or '2' for the cooldown. We wouldn't then need to update the localisation files if and when the cooldown changes.

Lastly, any new localisation content or any changes to existing text should be wrapped in specific comments like this:
```
; LWOTC Needs Translation
LocPromotionPopupText=...
; End Translation
```
This makes it easy for anyone that wants to contribute translations to find what needs translating. I like to support case-sensitive search as well, so please use the same capitalisation as in the example.